### ISSUE TITLE
Process XML comments in documentation comments

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#3097]
> As I said in my previous post, all comments would be supported, but only the XML tags will be recognized specially, unless we decide to add additional parsing of the comments.

Hi,

if I'm correct the doc comment support is now on the main branch. However I do not seem to be able to get the XML tag recognized. I followed the guidelines from [this page](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/summary) but when I hover on the function I documented using the <summary> tag, It displays everything as plain text, including the XML tags.

![vscode_doc](https://user-images.githubusercontent.com/1271675/51737493-a3f49d80-2084-11e9-96a7-464517c93111.png)

Did I misunderstand how XML tags are supposed to be parsed or am I missing something?

_Originally posted by @zupalex in https://github.com/Microsoft/vscode-cpptools/issues/399#issuecomment-457513013_

The issue tracking doxygen comments is https://github.com/Microsoft/vscode-cpptools/issues/658 .

**Comment by zupalex** [comment]
Thanks for moving that to the appropriate section.
So if I understand correctly, this feature is coming and was not supposed to work with the current doc comment support?

**Comment by sean-mcmanus** [comment]
We haven't decided yet on if/when this will be implemented -- more upvotes would help. I think doxygen support is higher priority.

**Comment by zupalex** [comment]
Oh I see. I just thought from one of your comment that it was supposed to be the plan, to supprot doc xml.
I don't eally mind, doxygen is fine with me as wll, as long as at least one of the 2 is parsed eventually :-)

**Comment by sean-mcmanus** [comment]
@zupalex Yeah, I thought doc xml would be automatically supported because VS does and we got the implementation from them, but it looks like adding the xml-stuff requires additional work.

**Comment by mklemarczyk** [comment]
@sean-mcmanus since the XML format is a native from Microsoft, I think is have higher priority. Especially if you are going to use VS Code for small and quick changes on the project that you are using Visual Studio normally, but it takes less to open VS Code and make a change than open Visual Studio. It would be very helpful.

**Comment by sean-mcmanus** [comment]
@mklemarczyk I believe our current plan is to start moving towards more code sharing so VS gets doxygen comment support and VS Code gets XML comment support as well.

**Comment by PoetaKodu** [comment]
Is there any progress on this feature?

**Comment by sean-mcmanus** [comment]
@PoetaKodu Not yet.

**Comment by frankierosa** [comment]
Any update? It will be good to prioritize these features, as these comments will help us many new c/c++ programmers.
